contracts/sol6/KyberNetworkProxy.sol

```diff
@@ -375,7 +375,11 @@ contract KyberNetworkProxy is
         uint256 srcAmount,
         address destAddress
     ) internal returns (UserBalance memory balanceBefore) {
-        require(src == ETH_TOKEN_ADDRESS || msg.value == 0, "msg.value should be 0");
+        if (src == ETH_TOKEN_ADDRESS) {
+            require(msg.value == srcAmount, "Sent eth != srcAmount");
```

Comment:
I think u mentioned etherscan doesn't show `!=` properly? Or was it `>` / `<`?

Reply:
fixed